tests/types/Engine.X-Ray.txt

```diff
@@ -0,0 +1 @@
+xrGame.dll
```

Comment:
you should add a test case of that file under a folder 
e.g: https://steamdb.info/depot/4501/

Reply:
oh ye forgot about it